cosmos/azure-spring-data-cosmos/cosmos-multi-tenant-samples/tenant-by-container/src/main/java/com/azure/spring/data/cosmos/example/Order.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.example;
+
+import com.azure.spring.data.cosmos.core.mapping.Container;
+import com.azure.spring.data.cosmos.core.mapping.PartitionKey;
+
+// Container names will be created/referenced dynamically using tenant id from TenantInterceptor,
+@Container(autoCreateContainer = false)
+public class Order {
+    private String id;
```

Comment:
Should we add the @Id tag to this field?

Reply:
yes, will add